cypress/integration/pages/index.js

```diff
@@ -62,6 +62,23 @@ const runCommonTests = ({ service, pageType }) => {
           .find('svg')
           .should('be.visible');
       });
+
+      if (appConfig[service].navigation) {
+        it('should have one visible navigation with a skiplink to h1', () => {
+          cy.get('nav')
+            .should('have.lengthOf', 1)
+            .should('be.visible')
+            .find('a[class^="SkipLink"]')
+            .should('have.lengthOf', 1)
+            .should('have.attr', 'href', '#content');
+          cy.get('nav a[class^="StyledLink"]')
```

Comment:
Not the biggest fan of `nav a[class^="StyledLink"]` as thats down to how styled components order their class names etc, can it be done differently?

Reply:
The class naming is due to how we've named the component itself, in conjunction with the use of styled components. These are unlikely to change, unless we have a major upgrade in styled components and even then it's unlikely for this naming convention to be broken. 

I think this is more resilient than finding the link via the DOM structure (nav > ul > li > 1).

Reply:
tbf I prefer `nav ul li a` its clear, specific and resiliant

Reply:
Also would `nav a` not work? What other `a` tags does the nav have?